README.md

```diff
@@ -68,29 +34,37 @@ Optional additional switches:
 
 Pass no arguments to `dg generateTestCases` to see additional switches and values.
 
-## Documentation
+## How it works
+
+There are 3 discrete primary elements:
+
+| Product | Description | Status | Notes |
+| ---- | ---- | ---- | ---- |
+| Profile | A representation of the data schema and rules which can be used to generate data | Stable - alpha | |
+| Generator | A tool for generating data from a given profile | Stable - alpha | Supports data generation and generation of data that has been [delibrately violated](./generator/docs/DeliberateViolation.md). |
+
+See [overview](./docs/Profiles.md) and [schema](./docs/Schema.md) for details about profiles and how they work.
 
-* [Data profiles](./docs/Profiles.md)
-* [Decision trees](./docs/DecisionTrees/DecisionTrees.md)
 * [Generator](./generator/README.md)
   * [Generation algorithm](./generator/docs/GenerationAlgorithm.md)
+  * [Set data generation process](./generator/docs/SetRestrictionAndGeneration.md)
+  * [Decision trees](./docs/DecisionTrees/DecisionTrees.md)
 
 * [Frequently asked questions](docs/FrequentlyAskedQuestions.md)
 
-Developers of Data Generator, or users who're curious about an aspect of its functionality, might be interested in the [key decisions log](docs/KeyDecisions.md).
+## Contributing
 
-## Development setup
+See [contributing](./.github/CONTRIBUTING.md) guidance and the [key decisions log](docs/KeyDecisions.md).
 
-1) Follow the [profiler setup instructions](profiler/README.md).
-2) Import the top level working directory `data-engineering-generator` into your IDE as an Maven project (In Eclipse: File > Import... > Maven > Existing Maven Project).  Four projects will be imported as a result:
-   - `parent`
-   - `generator`
-   - `profiler`
-   - `schemas` 
+### Development setup
 
-## Contributing
+This product requires **Java version 1.8** to be installed. Later versions are not supported at present.
+* Clone the repository
+* Follow the steps in the [setup instructions](./generator/README.md). The following need to be actioned for the data generator product, the remaining steps need to be actioned to be able to use the profiler.
+* Build the solution using your preferred method - the project is fully compatible with IntelliJ and Eclipse
+* Run the data generator or profiler - see the examples below.
 
-See [contributing](./.github/CONTRIBUTING.md)
+The product is confirmed compatible with Microsoft Windows, and should be compatible with all other operating systems, but this has not been confirmed to date.
```

Comment:
I don't think we need this disclaimer - although I'd encourage you to use Mac / Linux on your CI so that you can be comfortable that it does work across other OS

Reply:
Comment removed.